extras/rya.console/src/main/java/mvm/rya/shell/RyaConnectionCommands.java

```diff
@@ -164,8 +162,8 @@ public String connectToMongo(
             final MongoClient client = new MongoClient(new ServerAddress(hostname), Lists.newArrayList(creds));
 
             // Initialize the connected to Mongo shared state.
-            final MongoConnectionDetails accumuloDetails = new MongoConnectionDetails(username, password, instanceName, hostname);
-            final RyaCommands commands = RyaCommands.buildMongoCommands(accumuloDetails, client);
+            final MongoConnectionDetails mongoDetails = new MongoConnectionDetails(username, password, instanceName, hostname);
```

Comment:
because you can't connect to mongo without knowing which database to use.  the databases are named with instance names